public/app/features/folders/api/index.ts

```diff
@@ -0,0 +1,6 @@
+import { generatedAPI } from './endpoints.gen';
+
+export const folderAPI = generatedAPI.enhanceEndpoints({});
+
+// eslint-disable-next-line no-barrel-files/no-barrel-files
+export * from './endpoints.gen';
```

Comment:
can't we remove this, since we're already exposing the `folderAPI`? 🤔

Reply:
There are also types, but I can add a separate export for that. Would be nice to have all imports to be dome from the index file.